docs/rfcs/0005-coupons.md

```diff
@@ -0,0 +1,102 @@
+## Title for RFC
+
+coupons
+
+### Abstract
+
+This describes the various kinds of coupons and how they will be implemented.
+
+Note: For this RFC the "next" course run is the course run which the user would be enrolled
+in when they enroll in a course, whether it is currently running or will run soon.
```

Comment:
@noisecapella in what way this note is important ?

Reply:
When a coupon is for a course, that means it allows the user to pay for whatever the "next" course run is. The coupon would not work for any other run for that course.